lib/active_model/serializer/version.rb

```diff
@@ -2,6 +2,6 @@
 
 module ActiveModel
   class Serializer
-    VERSION = '0.10.11'.freeze
```

Comment:
please restore the freeze and I'll merge this

Reply:
there's a `frozen_string_literal: true` pragma available at the top of the file. The freeze is redundant here.

Reply:
oh, ok

Reply:
given the 2 facts,
- frozen_string_literal was introduced since Ruby 2.3
- AMS still supports Ruby 2.1 + 2.2

we might still need to have `.freeze`. correct me plz :)

Reply:
it's not super important. I'd have preferred it not be changed, since it's out of scope, but probably best to just merge this right now and formally support 6.1

Reply:
Should this have been changed to 0.10.12? Looks like [`0.10.11`](https://rubygems.org/gems/active_model_serializers/versions/0.10.11) already exists. 🙃

Reply:
Gem releases!= merged pr